src/components/contact/Contact.js

```diff
@@ -19,17 +20,26 @@ const inputs = [
   { type: "textarea", name: "message", placeholder: "Enter your message" },
 ];
 
+const FORMSPARK_FORM_ID = "X3j6PI8n";
+
 const Contact = () => {
   const { formData, handleChange, resetFormFields } = UseInput(initialValue);
   const [errors, setErrors] = useState({});
+  const [submit, submitting] = useFormspark({ formId: FORMSPARK_FORM_ID });
 
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
     e.preventDefault();
 
     const { newErrors, valid } = ValidateInput(formData);
 
     if (!valid) return setErrors(newErrors);
```

Comment:
does this line stop the form from attempting to submit anyway, or should we use a try/catch instead?

Reply:
Yes. will it work with a try/catch ?